subsys/bluetooth/audio/media_proxy.c

```diff
@@ -55,8 +55,9 @@ struct mprx {
 	struct media_player  local_player;
 #endif /* CONFIG_MCTL_LOCAL_PLAYER_CONTROL */
 
+	/* Control of remote players is currently only supported over Bluetooth, via MCC */
 #if defined(CONFIG_MCTL_REMOTE_PLAYER_CONTROL)
-	/* Remote media player - to have a player instance for the user, accessed via MCC */
+	/* Remote media player - to have a player instance for the user */
```

Comment:
This change in this commit does no longer seem to match the commit description.

Reply:
Commit removed (integrated into next).